webservice/webservice.py

```diff
@@ -291,22 +292,38 @@ def get_order():
     return jsonify({'order': order_list})
 
 
-@webservicebp.route('/repository/files/export', methods=['GET'])
+@webservicebp.route('/repository/files/export',
+                    methods=['GET'])
 def get_manifest():
     """
-    Creates and returns a manifest based on the filters pased on
-    to this endpoint
+    Creates and returns a manifest based on the filters and format passed on
+    to this endpoint.
+
     parameters:
         - name: filters
           in: query
           type: string
           description: Filters to be applied when generating the manifest
+          
+        - name: format
+          in: query
+          type: string
+          values: 'tsv' (default), 'bdbag'
+          description: Output format. Either TSV file or a compressed BDBag
+           send to S3 temporary storage.
     :return: A manifest that the user can use to download the files in there
     """
     # Setup logging
     logger = logging.getLogger("dashboardService.webservice.get_manifest")
+    # Create query parameters.
     filters = request.args.get('filters', '{"file": {}}')
     logger.debug("Filters string is: {}".format(filters))
+
+    # NOTE: "format" here is a HTTP query parameter, not a Python command.
+    format = request.args.get('format', '')
```

Comment:
You could just do `format = request.args.get('format', 'tsv')`, then get rid of the next two lines

Reply:
Good call. Done.